app/models/concerns/foreman_remote_execution/exportable.rb

```diff
@@ -0,0 +1,54 @@
+# This concern makes it easy to export an ActiveRecord object with specified
+# attributes and associations only.  If a specified assocation also includes
+# this concern, then it will likewise be exported. Custom attributes can be
+# specified as an options hast consisting of a custom export lambda.
+#
+# Example:
+#   attr_exportable :name, :address, :company => ->(user) { user.company.name }
+#
+module ForemanRemoteExecution
+  module Exportable
+    extend ActiveSupport::Concern
```

Comment:
Yes, although 1.12 is not so far away.  Did we decide on what our commitment is supporting older Foreman versions now that we're out of the super active development phase?  I have another PR that is 1.12-only (the new button helpers).